eventMacro.pl

```diff
@@ -92,10 +92,13 @@ sub commandHandler {
 		message 
 			"eventMacro MACRO: run macro MACRO\n".
 			"eventMacro list: list available macros\n".
-			"eventMacro status: shows current status\n".
-			"eventMacro stop: stop current macro\n".
-			"eventMacro pause: interrupt current macro\n".
-			"eventMacro resume: resume interrupted macro\n".
+			"eventMacro macro_status: shows current macro status\n".
```

Comment:
I like 'macro_status|stop|pause|resume' but I also think that the automacro commands got way bigger than I would like them to be. Maybe 'automacro_pause|resume|status'?